it/v1/endpoints/CreateBFLossControllerISpec.scala

```diff
@@ -117,6 +117,7 @@ class CreateBFLossControllerISpec extends IntegrationBaseSpec {
     "return 500 (Internal Server Error)" when {
 
       createErrorTest(Status.BAD_REQUEST, "INVALID_PAYLOAD", Status.INTERNAL_SERVER_ERROR, DownstreamError)
+      createErrorTest(Status.BAD_REQUEST, "INVALID_XXX", Status.INTERNAL_SERVER_ERROR, DownstreamError)
```

Comment:
I think it might be nice for testing for errors we don't expect to receive from DES that we appropriately name them.  Either comment the test, or name the DES error code something that explains it.